lib/vtt.js

```diff
@@ -1204,8 +1204,17 @@
     })();
   };
 
-  WebVTT.Parser = function(window, decoder) {
+  WebVTT.Parser = function(window, vttjs, decoder) {
```

Comment:
Hmm, yeah. I'm not too partial to any name. Just `vttjs` doesn't make me think of shims particularly for VTTCue or VTTRegion that aren't the built in ones.